src/The_Cartridge_Header.md

```diff
@@ -127,145 +137,160 @@ Code | Publisher
- $FC | POCKET CAMERA
- $FD | BANDAI TAMA5
- $FE | HuC3
- $FF | HuC1+RAM+BATTERY
+## 0147 – Cartridge Type
+
+This byte indicates what kind of Memory Bank Controller (and other hardware)
+the cartridge contains.
+
+Code  | Type
+------|------
+`$00` | ROM ONLY
+`$01` | MBC1
+`$02` | MBC1+RAM
+`$03` | MBC1+RAM+BATTERY
+`$05` | MBC2
+`$06` | MBC2+BATTERY
+`$08` | ROM+RAM [^rom_ram]
+`$09` | ROM+RAM+BATTERY [^rom_ram]
+`$0B` | MMM01
+`$0C` | MMM01+RAM
+`$0D` | MMM01+RAM+BATTERY
+`$0F` | MBC3+TIMER+BATTERY
+`$10` | MBC3+TIMER+RAM+BATTERY [^mbc30]
+`$11` | MBC3
+`$12` | MBC3+RAM [^mbc30]
+`$13` | MBC3+RAM+BATTERY [^mbc30]
+`$19` | MBC5
+`$1A` | MBC5+RAM
+`$1B` | MBC5+RAM+BATTERY
+`$1C` | MBC5+RUMBLE
+`$1D` | MBC5+RUMBLE+RAM
+`$1E` | MBC5+RUMBLE+RAM+BATTERY
+`$20` | MBC6
+`$22` | MBC7+SENSOR+RUMBLE+RAM+BATTERY
+`$FC` | POCKET CAMERA
+`$FD` | BANDAI TAMA5
+`$FE` | HuC3
+`$FF` | HuC1+RAM+BATTERY
 
 [^rom_ram]:
-No licensed cartridge makes use of this option. Exact behaviour is unknown.
+No licensed cartridge makes use of this option. The exact behavior is unknown.
 
 [^mbc30]:
-MBC3 with RAM size 64 KByte refers to MBC30, used only in _Pocket Monsters Crystal Version_ for Japan.
-
-## 0148 - ROM Size
-
-Specifies the ROM Size of the cartridge. Typically calculated as "N such that 32 KiB
-<< N".
-
-Code | Size      | Amount of banks
------|-----------|-----------------
- $00 |  32 KByte | 2 (No ROM banking)
- $01 |  64 KByte | 4
- $02 | 128 KByte | 8
- $03 | 256 KByte | 16
- $04 | 512 KByte | 32
- $05 |   1 MByte | 64
- $06 |   2 MByte | 128
- $07 |   4 MByte | 256
- $08 |   8 MByte | 512
- $52 | 1.1 MByte | 72 [^weird_rom_sizes]
- $53 | 1.2 MByte | 80 [^weird_rom_sizes]
- $54 | 1.5 MByte | 96 [^weird_rom_sizes]
+MBC3 with a 64 KiB RAM refers to MBC30, used only in
+_Pocket Monsters: Crystal Version_ (the Japanese version of
+_Pokémon Crystal Version_).
+
+## 0148 – ROM Size
+
+This byte specifies the size of the ROM on the cartridge. In most cases the ROM size
+is given by `32 KiB x (2 ^ <value>)`:
+
+Code  | ROM size  | Number of ROM banks
+------|-----------|-----------------
+`$00` |  32 KiB   | 2 (no banking)
+`$01` |  64 KiB   | 4
+`$02` | 128 KiB   | 8
+`$03` | 256 KiB   | 16
+`$04` | 512 KiB   | 32
+`$05` |   1 MiB   | 64
+`$06` |   2 MiB   | 128
+`$07` |   4 MiB   | 256
+`$08` |   8 MiB   | 512
+`$52` | 1.1 MiB   | 72 [^weird_rom_sizes]
+`$53` | 1.2 MiB   | 80 [^weird_rom_sizes]
+`$54` | 1.5 MiB   | 96 [^weird_rom_sizes]
 
 [^weird_rom_sizes]:
 Only listed in unofficial docs. No cartridges or ROM files using these sizes are known.
-As the other ROM sizes are all powers of 2, these are likely inaccurate. The source for these
+As the other ROM sizes are all powers of 2, these are likely inaccurate. The source of these
 values is unknown.
 
-## 0149 - RAM Size
+## 0149 – RAM Size
 
-Specifies the size of the external RAM in the cartridge (if any).
+This byte specifies the size of the RAM on the cartridge (if any).
 
```

Comment:
```suggestion

If the [cartridge type](<#0147 – Cartridge Type>) does not include "RAM" in its name, this should be set to 0.
This includes MBC2, since its 512 × 4 bits of memory are built directly into the mapper.

```
...and then the footnote below would be removed.
Then, we'd also delete the MBC2 footnote.